Sources/OktaIdxAuth/OktaIdxAuth.swift

```diff
@@ -91,6 +92,24 @@ import OktaIdx
         }
     }
     
+    @objc(OktaIdxAuthSocialAuthentication)
+    public class SocialAuth: NSObject {
+        @available(iOSApplicationExtension 13.0, *)
+        @objc(OktaIdxAuthSocialAuthenticationOptions)
+        public class Options: NSObject {
+            @objc
+            public let presentationContext: ASWebAuthenticationPresentationContextProviding
```

Comment:
I would like to use one object and not separate iOS12 class. But Swift doesn't support stored property with `@available` attribute. The error is thrown.